tests/ProtoTests/BasicPaginated/Gapic/BasicPaginatedGapicClient.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+declare(strict_types=1);
+
+namespace Testing\BasicPaginated\Gapic;
+
+use Google\ApiCore\ApiException;
+use Google\ApiCore\CredentialsWrapper;
+use Google\ApiCore\GapicClientTrait;
+use Google\ApiCore\RetrySettings;
+use Google\ApiCore\Transport\GrpcTransport;
+use Google\ApiCore\Transport\RestTransport;
+use Google\ApiCore\Transport\TransportInterface;
+use Google\ApiCore\ValidationException;
+use Google\Auth\FetchAuthTokenInterface;
+use Testing\BasicPaginated\Request;
+use Testing\BasicPaginated\Response;
+
+/**
+ *
+ * This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * ```
+ * $basicPaginatedServiceClient = new BasicPaginatedClient();
+ * try {
+ *     // Iterate over pages of elements
+ *     $pagedresponse = $basicPaginatedServiceClient->methodPaginated();
+ *     foreach ($pagedresponse->iteratePages() as $page) {
+ *         foreach ($page as $element) {
+ *             // doSomethingWith($element);
+ *         }
+ *
```

Comment:
Can you remove this blank line easily?

Reply:
No, unfortunately it's not trivial to remove this.
I will revisit all the formatting issues in the generated code at a later date.